helm/cluster/values.schema.json

```diff
@@ -1929,6 +1929,12 @@
                             "description": "Flag that indicates if the Cilium HelmRelease is enabled and templated. This is meant only for the initial development purposes for the sake of incrementally integrating cluster chart into cluster-$provider apps.",
                             "default": false
                         },
+                        "cleanupHelmReleaseResourceEnabled": {
```

Comment:
nit: cleanupHelmReleaseResource**s**Enabled instead of cleanupHelmReleaseResourceEnabled 😬